tgui/packages/tgui/interfaces/AntagInfoHeretic.tsx

```diff
@@ -219,63 +231,102 @@ const InformationSection = (props) => {
+                      <Button
+                        key={node.name}
+                        color="transparent"
+                        tooltip={`${node.name}:
+                          ${node.desc}`}
+                        onClick={
+                          node.disabled || node.finished
+                            ? undefined
+                            : () => act('research', { path: node.path })
+                        }
+                        width={node.ascension ? '192px' : '64px'}
+                        height={node.ascension ? '192px' : '64px'}
+                        m="8px"
+                        style={{
+                          borderRadius: '50%',
+                        }}
+                      >
+                        <DmIcon
+                          icon="icons/ui_icons/antags/heretic/knowledge.dmi"
+                          icon_state={
+                            node.disabled
+                              ? 'node_locked'
+                              : node.finished
+                                ? 'node_finished'
+                                : node.bgr
+                          }
+                          height={node.ascension ? '192px' : '64px'}
+                          width={node.ascension ? '192px' : '64px'}
+                          top="0px"
+                          left="0px"
+                          position="absolute"
+                        />
+                        <DmIcon
+                          icon={node.icon_params.icon}
+                          icon_state={node.icon_params.state}
+                          frame={node.icon_params.frame}
+                          direction={node.icon_params.dir}
+                          movement={node.icon_params.moving}
+                          height={node.ascension ? '152px' : '64px'}
+                          width={node.ascension ? '152px' : '64px'}
+                          top={node.ascension ? '20px' : '0px'}
+                          left={node.ascension ? '20px' : '0px'}
+                          position="absolute"
+                        />
+                        <Box
+                          position="absolute"
+                          top="0px"
+                          left="0px"
+                          backgroundColor="black"
+                          textColor="white"
+                          bold
+                        >
+                          {node.finished
+                            ? ''
+                            : node.cost > 0
+                              ? node.cost
+                              : 'FREE'}
+                        </Box>
+                      </Button>
+                      {node.ascension ? (
+                        <Box textAlign="center" fontSize="32px">
+                          <span style={hereticPurple}>DUSK</span>
+                        </Box>
+                      ) : (
+                        ''
+                      )}
```

Comment:
doesn't seem to need the `''` here. perhaps just 
```tsx
{!!node.ascension && (<Box>...</Box>)}
```

Reply:
I can also do this with the other redundant ternary above